cytominer_eval/evaluate.py

```diff
@@ -8,7 +8,7 @@
 from typing import List, Union
 
 from cytominer_eval.transform import metric_melt
-from cytominer_eval.transform.util import check_replicate_groups
+from cytominer_eval.utils.transform_utils import check_replicate_groups
```

Comment:
For me, this line gives an error because utils are not imported here or in the init.py file. But I am unsure why the tests don't give that error? 

This applies to all imports of util below

Reply:
oh interesting, what is the error?

Is it possible that your testing environment is out-of-date?

I also added a `__init__.py` file inside `cytominer_eval/utils` - so this might help. It should have been there in the first place, so thanks for making this note.

Reply:
Yea I know it works, for some reason pycharm is complaining about it. I think it's because if you run that file on its own, it won't find the util. 
but all good